auth/api/http/policies/responses.go

```diff
@@ -39,7 +39,7 @@ func (res deletePoliciesRes) Headers() map[string]string {
 }
 
 func (res deletePoliciesRes) Empty() bool {
-	return false
+	return true
```

Comment:
Why is this changed?

Reply:
Since `deleteResponse` has no body, `Empty` should return `false`.

Reply:
You introduced the change where it returns `true`. I am confused now - what should the return value be - `false` as it is today, or `true` as you propose.

Reply:
My bad, it is empty, so it should return true.

Reply:
OK, good - resolving this one.